web/playwright.config.ts

```diff
@@ -7,7 +7,7 @@ const config: PlaywrightTestConfig = {
   use: {
     viewport: { width: 1280, height: 720 },
     acceptDownloads: true,
-    baseURL: "http://localhost:4173",
+    baseURL: "https://ris.dev.ds4g.net",
```

Comment:
Does this mean, we cannot run e2e tests locally? Wouldn't it be nice, to get the baseUrl from env variables?

Reply:
Exactly. The problem is the cloud bucket. Our S3ClientMock doesn't do anything with the uploaded docx file.
The only way for now is to run the backend against staging. 
But yes, we can put the baseUrl into an env variable.

Reply:
I also think it would be nice to be able to run them locally. It might only work though if we run the profile `staging` locally and not actually the `local` profile? Because we need the real AWS client and not the mockup one 🤔

Reply:
Also, so far I had tried to keep at least the staging url private, there's a secret for this, which is being populated via environment variable for the journey tests here: https://github.com/digitalservicebund/ris-backend-service/blob/56519668ca152bda8c160abba480f700b14a85b6/src/test/resources/application.properties#L1

Reply:
done